doc/Reference.tex

```diff
@@ -692,6 +692,14 @@ \section{Plot widget}
 \end{center}
 \end{description}
 
+\section{Sheet Widget}
+ \label{Sheets}\label{Item} The Sheet widget displays input data as a
+ number, rather than as a 2D graph, as in the case of the plot widget.
+ To use the Sheet widget, simply wire a variable or other item on your
+ canvas to the left-hand side of the sheet widget box. This will diplay
+ the input data as a number. Note that only one wire can be connected to
+ a sheet, as the sheet can only display a single input value.
+
```

Comment:
Actually, it can display rank 0, 1 and rank 2 tensors. Try creating a parameter, and set the initial condition to rand(3,5) (see section on tensor initial conditions), and wire that into a sheet.